.bazelrc

```diff
@@ -22,6 +22,8 @@ startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
 build --flag_alias=bazel_code_coverage=//build/toolchains:bazel_code_coverage_flag
 build --flag_alias=crdb_test=//build/toolchains:crdb_test_flag
 build --flag_alias=crdb_test_off=//build/toolchains:crdb_test_off_flag
+build --flag_alias=crdb_bench=//build/toolchains:crdb_bench_flag
+build --flag_alias=crdb_bench_off=//build/toolchains:crdb_bench_off_flag
```

Comment:
There doesn't seem to be a reason for this `crdb_bench_off` thing, it looks like you can remove it. `crdb_test_off` is there for a reason (to override `crdb_test` if you need to run it in a circumstance where it's implied, like `bazel test`)